public/locations-list-map.js

```diff
@@ -982,6 +990,26 @@ function centerMapToBounds(map, bounds, maxZoom) {
   }
 }
 
+const translateEnumValue = (value) => {
+  const enumValue = ENUM_MAPPINGS[value.toLowerCase()];
+
+  if (enumValue) {
+    return $.i18n(enumValue.name);
+  }
+
+  return value;
+};
+
+const translateEnumList = (enumListString) => {
+  if (enumListString) {
+    return enumListString.split(', ').map((stringValue) => (
+      translateEnumValue(stringValue)
+    )).join(', ')
```

Comment:
I'd tend to `split(',').map(s => translateEnumValue(s.trim()))` to make it robust to whitespace mistakes. But this is fine too.